net/strongswan/files/swanctl.init

```diff
@@ -219,6 +219,7 @@ config_child() {
 	local mode="$2"
```

Comment:
The description of the commit is not correct.

The "connections.<conn>.children.<child>.interface" key in swanctl sets the interface of the policy. It does not influence routing.

This means that the policy will have the additional criteria of the outgoing interface being the value of this setting.
Means, you can potentially have different policies (and thus states) for different outgoing interfaces. That can enable routing of traffic going out of different interfaces to different peers.

Reply:
Fixed.